src/main/java/frc/robot/subsystems/shooter/ShooterInterpolation.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.math.interpolation.InterpolatingDoubleTreeMap;
+
+public class ShooterInterpolation {
```

Comment:
Nit on naming: all the interpolating, that's an implementation detail. This class converts distance into speeds, but the callers don't care how. They'd be just as happy if you computed values from a continuous curve. 

I only point it out because I think people get hung up on "interpolation" as this big, important thing.

Reply:
This is a good point, the users of this class just want the values computed, they don't care how. And it doesn't necessarily need to be interpolation. 

Therefore, I think naming this class with a name that shows what the values outputted from the class actually mean rather than how the class actually computes those values would be better. Maybe something such as `AimingValuesInterpolator` or something.